src/test/java/uk/nhs/ctp/service/TriageServiceTest.java

```diff
@@ -140,4 +122,34 @@ public void testQuestionnaireRequestNotMadeWhenOnlyResultIsPresent() throws FHIR
     verify(cdssService, times(0)).getQuestionnaire(anyLong(), anyString());
   }
 
+  @Test
+  public void processTriageRequest_withCarePlanIds_willCompleteCarePlans() throws Exception {
+    var carePlanIds = new String[]{"id1", "id2"};
+    when(mockCdssRequestDTO.getCarePlanIds()).thenReturn(carePlanIds);
+
+    when(mockCdssResult.hasResult()).thenReturn(true);
+    when(mockCdssResult.hasQuestionnaire()).thenReturn(false);
+    when(responseService.buildResponse(mockCdssResult, null, 1L, 1L))
+        .thenReturn(mockCdssResponseDTO);
```

Comment:
The tests seem to pass when these aren't enabled so as per comment above suggest highly simplifying.

Reply:
Agreed, will do!